src/components/contributorDashboard.js

```diff
@@ -0,0 +1,320 @@
+import React from "react";
+import { nav } from "react-bootstrap";
+import "../dashboard.css";
+
+function AsideNav() {
+  return (
+    <nav className="asideNav col-3 px-5 pt-3" style={{ background: "#344127" }}>
+      <div className="container text-left">
+        <a className="h2 text-white Aside-title" href="/">
+          <img src="images/onboard_menu_icon.svg" alt="" /> Open Data
+        </a>
+
+        <ul className="navbar-nav mt-5 ">
+          <li className="nav-item active">
+            <a href="/" className="nav-link h4">
+              <i className="fas fa-home mr-2"></i> Dashboard
+            </a>
```

Comment:
Hello Taro, nice work.
Suround the <a> tags with <Link to ="/your url path"> </Link>  from react-router-dom for it to work well in production, I experienced the problem.
example  <Link><a href="/" className="nav-link h4">
              <i className="fas fa-home mr-2"></i> Dashboard
            </a></Link>
Remember to import Link from "react-router-dom";